src/components/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,100 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import {isBrowser} from 'utils/executionEnvironment';
+
+import DropDownSelector from './DropDownSelector';
+import DropDownItems from './DropDownItems';
+import styles from './index.scss';
+
+export default class DropDown extends Component {
+  static propTypes = {
+    align: PropTypes.oneOf(['left', 'right']),
```

Comment:
I think `horizontalPosition` is a bit different than alignment because it means dropdown will open to your left or right, like this:

![screen shot 2019-01-02 at 16 07 09](https://user-images.githubusercontent.com/36070274/50600266-7dd64600-0ea8-11e9-97c7-aff4d62c8ac5.png)
![screen shot 2019-01-02 at 16 07 13](https://user-images.githubusercontent.com/36070274/50600267-7dd64600-0ea8-11e9-824a-bb5de2827327.png)

Reply:
yeah, that's how I would understand horizontalPosition the positioning along the horizontal axis so from left to right. How do you understand it?

Reply:
I understood it as what `react-boostrap` calls `pullRight`

Reply:
I didn't added any left or right opening option because it isn't necessary anywhere on our usage (yet).